test/built-ins/TypedArray/prototype/values/make-in-bounds-after-exhausted.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2024 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.values
+description: >
+  Iterator is still exhausted when typedarray is changed to in-bounds.
+features: [TypedArray, resizable-arraybuffer]
```

Comment:
Just a suggestion, to indicate which step is being tested.

```suggestion
info: |
  %TypedArray%.prototype.values ( )

  ...
  3. Return CreateArrayIterator(O, VALUE).
  
features: [TypedArray, resizable-arraybuffer]
```

Reply:
I didn't add spec step references here, because the relevant steps are part of [CreateArrayIterator](https://tc39.es/ecma262/#sec-createarrayiterator), but it doesn't make sense to refer to `CreateArrayIterator`, because the current spec text is incorrect and has to be rewritten: https://github.com/tc39/ecma262/issues/3136

Reply:
OK, makes sense not to add the reference here, then.